moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1813,16 +1780,17 @@ bool RobotState::setFromIKSubgroups(const JointModelGroup* jmg, const EigenSTL::
     ik_queries[i].orientation.w = quat.w();
   }
 
-  if (attempts == 0)
-    attempts = jmg->getDefaultIKAttempts();
-
   // if no timeout has been specified, use the default one
   if (timeout < std::numeric_limits<double>::epsilon())
     timeout = jmg->getDefaultIKTimeout();
+  ros::WallTime start = ros::WallTime::now();
 
   bool first_seed = true;
-  for (unsigned int st = 0; st < attempts; ++st)
+  unsigned int attempts = 0;
```

Comment:
Why not remove `attempts` entirely?

Reply:
Just to print the attempt in the following debug message.

Reply:
That seems unnecessary since there will only ever be a single attempt

Reply:
I hadn't seen that. good point